src/Scheduler.js

```diff
@@ -0,0 +1,79 @@
+import './SchedulerJob'
+
+import later from 'later'
+
+export class Scheduler {
+
+	provider = later
+	cli = null
+
+	jobs = [ ]
+
+	constructor(cli) {
+		this.cli = cli
+	}
+
+	start() {
+		return new Promise((resolve, reject) => {
+			this.jobs.map((job) => {
```

Comment:
Looks like there’s a bug here that could cause reject to fire multiple times… Let’s refactor to:

```js
for(const job of jobs) {
	if(!job.schedule) {
		return reject('Job is missing a schedule.')
	}

	job.start()
}
```